README.md

```diff
@@ -41,13 +41,16 @@ then you will want to execute in 2 separate consoles:
 frontend
 ```bash
 cd frontend
-npm run dev
+pnpm run dev
 ```
 backend
 ```bash
 cd backend/LexBoxApi
 dotnet watch
 ```
+
+If you don't have pnpm installed, run `npm install -g pnpm`.
```

Comment:
Actually it would be better to suggest using corepack to install it, like this `corepack prepare pnpm@latest --activate` as suggested by their documentation. Core pack is a feature in more recent node versions that help you manage other package managers.